packages/web-components/src/components/ic-side-navigation/__snapshots__/ic-side-navigation.spec.ts.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ic-side-navigation renders expanded: renders-expanded 1`] = `
-<ic-side-navigation app-title="ACME" class="sm-expanded" expanded="true">
+<ic-side-navigation app-title="ACME" class="sm-expanded" expanded="true" style="--sm-side-navigation-bottom-bar-height: undefinedpx;">
```

Comment:
Is this meant to come through as undefinedpx or should it have a default value if it's not been set?

Reply:
I've set a default value for it in the CSS file (on line 18). Not sure if it's because I've set the default as a rem value